Automattic/studio
# fix(assistant): number of monthly prompts

## Related issues

- https://github.com/Automattic/dotcom-forge/issues/9836

## Proposed Changes
We have Assistant in Studio. We give 200 prompts per months and we store the number of used prompts at backend and retrieve it from there.
When a user is offline - we don't know the number of used prompts so we render 0/200 (on screenshot is 1000 since we set this number for A12S). But it doesn';t make sense to render `0`, so with this PR, we are changing it to render "offline" message(look test instructions).


## Testing Instructions
1. Log in to Studio
2. Click on you avatar in top right corner to open "Settings"
3. Turn on WiFi
4. Assert that you see message about offline mode

<table>
<tr>
<td>BEFORE
<td>AFTER
</tr>
<tr>
<td><img width="312" alt="Screenshot 2024-11-15 at 10 55 49" src="https://github.com/user-attachments/assets/1401015b-0555-4db1-b874-028ebc137e98">
<td><img width="314" alt="Screenshot 2024-11-15 at 10 46 34" src="https://github.com/user-attachments/assets/f8a60707-d3ff-4d4c-bdf6-93e7a6f4eb6c">
</tr>
</table>
